src/Arrays/MPIStateArrays.jl

```diff
@@ -1,13 +1,13 @@
-module AtmosStateArrays
+module MPIStateArrays
 using LinearAlgebra
 
 using MPI
 
-export AtmosStateArray
+export MPIStateArray
 
 """
-    AtmosStateArray{S <: Tuple, T, DeviceArray, N,
-                    DATN<:AbstractArray{T,N}, Nm1, DAI1} <: AbstractArray{T, N}
+    MPIStateArray{S <: Tuple, T, DeviceArray, N,
+                  DATN<:AbstractArray{T,N}, Nm1, DAI1} <: AbstractArray{T, N}
```

Comment:
Happy to keep this for the time being, but I was wondering if we could come up with a better name for this. Maybe `MPIGhostArray` or something like that?

Reply:
Not sure what the word `Ghost` means in this context. I tend to think of `Ghost` being data that an mpi rank depends another ran owns.

Reply:
But FYI, I'm not wed to this name... just had to name it something other than `AtmosStateArray`. @lcw might have thoughts.

Reply:
Let's leave it as is for now.